tools/repo_setup.py

```diff
@@ -369,6 +381,55 @@ def update_repo(self, submodule):
         else:
             print_error("Error loading submodule info")
 
+    def get_archives(self, archive):
+        archive_info = self.get_info_by_archive(archive)
+
+        if archive_info:
+            organization, project, artifact, version, path = archive_info
+
+            full_repo_path = os.path.join(self.script_dir, path)
```

Comment:
Assim evita carregar a identação pro bloco todo ali

```suggestion
        if not archive_info:
            print_error("Error loading archive info")
            return
        
        organization, project, artifact, version, path = archive_info

        full_repo_path = os.path.join(self.script_dir, path)
```